ios/Capacitor/Capacitor/CAPBridgeDelegate.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-internal protocol CAPBridgeDelegate: AnyObject {
+public protocol CAPBridgeDelegate: AnyObject {
```

Comment:
I really want to get rid of this class, for cap 6 maybe? I don't really see the point of it. We should at least change the name, it's not a delegate.

Reply:
Yeah, and it was also internal prior to this tbh

Reply:
@markemer you want to make that change in a separate PR?

Reply:
I think we actually might just need to rename it, I'm looking into how we use this now.